.circleci/config.yml

```diff
@@ -108,6 +109,13 @@ jobs:
             ls ~/package/
             ghr -t ${GITHUB_TOKEN_FOR_RELEASES} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ~/package/
 
+            echo $DOCKER_HUB_TOKEN | docker login -u thomaseizinger --password-stdin
```

Comment:
Seriously? You can't have a generic token for a repo not associated to a specific user?? :(

Reply:
Don't think so. Up until September you couldn't even do tokens :sweat_smile: